lighthouse-core/config/default.js

```diff
@@ -27,6 +27,18 @@ module.exports = {
   {
     "passName": "redirectPass",
     "useThrottling": false,
+    // Speed up the redirect pass by blocking stylesheets, fonts, and images
+    "blockedUrlPatterns": [
+      "*.css",
```

Comment:
formatting nit:  this is pretty low signal info, compared to the rest of config. 
can these all share a line?